pyproject.toml

```diff
@@ -41,7 +41,7 @@ dependencies = [
     "numpy<2.0.0",
     "netcdf4<=1.6.4",
     "h5netcdf>=1.1.0",
-    "openeo>=0.29.0",
+    "openeo>=0.31.0",
```

Comment:
I'll wait for this ticket to close before changing the OpenEO version

https://github.com/Open-EO/openeo-gfmap/issues/151

Reply:
ok but we can't merge this PR without bumping the version because the new slope collection cannot be loaded with `0.29.0`. Something in the python client changed and the former version cannot work with it.

Reply:
Okay, I'll try to close this issue GFMAP today

Reply:
@kvantricht Nevermind sorry! this issue will come with Version 0.32 and the feature is not yet merged on the client repository 

- https://github.com/Open-EO/openeo-python-client/pull/607